pkg/shell/hosts.jsx

```diff
@@ -127,6 +127,7 @@ export class CockpitHosts extends React.Component {
     filterHosts(host, term) {
         if (!term)
             return host;
+        console.log("A added line");
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17475-20220628-064306-0d71ea51-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)